go/src/koding/klientctl/DEPLOYMENT.md

```diff
@@ -0,0 +1,63 @@
+klient deployment notes
+=======================
+
+* Configure
+
+Configure an AWS profile for uploading files to S3:
+
+```bash
+~ $ aws --profile koding-klient configure
+AWS Access Key ID [None]: ***
+AWS Secret Access Key [None]: ***
+Default region name [None]: us-east-1
+Default output format [None]: json
+```
+
+Export segment.io key:
+
+```bash
+~ $ export KD_SEGMENTIO_KEY=***
```

Comment:
[metrics-client.go#L43](https://github.com/koding/koding/blob/master/go/src/koding/klientctl/metrics/metrics-client.go#L43)

looks like segment.io is used for collecting metrics

(/cc @leeola)